src/administration/snapshot-and-restore.md

```diff
@@ -42,25 +41,51 @@ Finding snapshots for the environment master
 ```
 
 You can restore a specific snapshot with the CLI as follows:
+
 ```bash
 $ platform snapshot:restore 92c9a4b2aa75422efb3d
 ```
 
 > **note**
 > You need "admin" role to restore your environment from a snapshot.
 
-### Automated snapshots
+## Automated snapshots
 
-No snapshot is triggered automatically on Platform.sh Standard. You can
-trigger your snapshot via the Web Interface or via the CLI.
+No snapshot is triggered automatically on Platform.sh Standard. You can trigger your snapshot via the Web Interface or via the CLI.
 
-If you want to automate your snapshots, you can use Jenkins and trigger
-with the CLI command as follows:
+Snapshots may be triggered from an automated system, such as cron, Jenkins, or another CI service by calling the CLI.  That requires first obtaining an [API token for authentication](/gettingstarted/cli.md#api-tokens).  Add the token to your Platform.sh project as listed there.
 
-```bash
-$ platform snapshot:create --yes --no-wait
+Now add a build hook to your `.platform.app.yaml` file to download the CLI as part of the build process.  
+
+```yaml
+hooks:
+    build: |
+        curl -sS https://platform.sh/cli/installer | php
 ```
 
+That will download the CLI to a known directory within the `.platform` directory, which is available in the path.  Because the API Token is available it will now be able to run authenticated commands against any project available to the user who created the token.
```

Comment:
This will download the CLI to a known directory, `.platformsh/bin`, which will be added to the `PATH` at runtime (via the `.environment` file). Because the API token is available, the CLI will now be able to run authenticated commands, acting as the user who created the token.